app/assets/images/strava.svg

```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg" width="2500" height="2500"><path d="M0 0h16v16H0z" fill="#fc4c02"/><g fill="#fff" fill-rule="evenodd"><path d="M6.9 8.8l2.5 4.5 2.4-4.5h-1.5l-.9 1.7-1-1.7z" opacity=".6"/><path d="M7.2 2.5l3.1 6.3H4zm0 3.8l1.2 2.5H5.9z"/></g></svg>
```

Comment:
We should be using the font-awesome strava icon here: https://fontawesome.com/icons/strava

Reply:
Note: this may conflict with the mockup. That is ok. We can address the _why_ that looks different when we present to @runbeyond before merging.

Reply:
When shall we have a look?